cinema/serializers.py

```diff
@@ -74,7 +121,55 @@
 class MovieSessionDetailSerializer(MovieSessionSerializer):
     movie = MovieListSerializer(many=False, read_only=True)
     cinema_hall = CinemaHallSerializer(many=False, read_only=True)
+    taken_seats = serializers.SerializerMethodField()
```

Comment:
The `get_taken_seats` method correctly returns a list of taken seats, but verify that the `tickets` relation on `MovieSession` is correctly set up to return the expected results.